styles/footer.module.scss

```diff
@@ -0,0 +1,63 @@
+.footer {
```

Comment:
We can keep this file, but please pare it down to the bare minimum.

Reply:
done

Reply:
Bump, please remove any unused styles in this file.